extensions/interactions/DragAndDropSortInput/DragAndDropSortInput.py

```diff
@@ -49,18 +52,14 @@ class DragAndDropSortInput(base.BaseInteraction):
                 'min_value': 1
             }],
             'items': {
-                'type': 'html',
-                'ui_config': {
-                    'hide_complex_extensions': True,
-                    'placeholder': 'Enter an option for the learner to drag' +
```

Comment:
What happened to the placeholder and the RTE extension customizations?

/cc @vibhor98

Reply:
I didn't find this is being used anywhere.

@vibhor98, is it used somewhere?

Reply:
Yes, the placeholder and RTE extension customisations are used in Customise Interaction modal that is used while adding an interaction to the exploration. Please don't remove this!

Reply:
Oh, I just found out that placeholders are no longer rendered in RTEs. Is it due to RTE migration to CKEditor (not sure!) @bansalnitish  @AllanYangZhou? Previously everything was fine.